src/memory_service/fam_memory_service_server.cpp

```diff
@@ -90,6 +90,7 @@ Fam_Memory_Service_Server::Fam_Memory_Service_Server(uint64_t rpcPort,
 }
 
 void Fam_Memory_Service_Server::run() {
+    memoryService->init_atomic_queue();
```

Comment:
we should call init_atomic_queue from memory server direct. cis will not always do rpc path to memory server. It can call the memory server direct without calling memory service server